command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
+That executable MUST support commands with the following template:
+
+```sh
+$ funC [global-options] <COMMAND> [command-specific-options] <command-specific-arguments>
+```
+
+## Global options
+
+None are required, but the runtime MAY support options that start with at least one hyphen.
+Global options MAY take positional arguments (e.g. `--log-level debug`).
+Command names MUST NOT start with hyphens.
+The option parsing MUST be such that `funC <COMMAND>` is unambiguously an invocation of `<COMMAND>` (even for commands not specified in this document).
+If the runtime is invoked with an unrecognized command, it MUST exit with a nonzero exit code and MAY log a warning to stderr.
+Beyond the above rules, the behavior of the runtime in the presence of commands and options not specified in this document is unspecified.
+
+## Character encodings
+
+This API specification does not cover character encodings, but runtimes SHOULD conform to their native operating system.
+For example, POSIX systems define [`LANG` and related environment variables][posix-lang] for [declaring][posix-locale-encoding] [locale-specific character encodings][posix-encoding], so a runtime in an `en_US.UTF-8` locale SHOULD write its [state](#state) to stdout in [UTF-8][].
+
+## Commands
+
+### create
+
+[Create][create] a container from a [bundle directory][bundle].
```

Comment:
On Thu, Jul 28, 2016 at 11:54:26AM -0700, Doug Davis wrote:
> > +[Create][create] a container from a [bundle directory][bundle].
> 
> Let's add one line to each command showing usage syntax - e.g.
> ```
> funC [ options ] create ID
> ```
> When all you care about (quickly) is the syntax of the command, this
> is much easier to read.

So:

  funC [global-options] create [command-specific-options] <ID>

Or:

  funC [global-options] create [--bundle <PATH>] [--pid-file <PATH>] <ID>

I'm ok with that, although I think “how do I use this quickly” is
better answered by the example sections [1].

[1]: https://github.com/opencontainers/runtime-spec/pull/513/files/7604b096ef979f0d559165e78f51a54e3e9536eb#diff-0c1e5a5baa5649945697a530e09acbf5R46